core/new-gui/src/app/dashboard/user/component/user-dataset/user-dataset-explorer/user-dataset-version-creator/user-dataset-version-creator.component.ts

```diff
@@ -0,0 +1,174 @@
+import { Component, EventEmitter, Input, OnInit, Output } from "@angular/core";
+import { FormBuilder, FormGroup, Validators } from "@angular/forms";
+import { FormlyFieldConfig } from "@ngx-formly/core";
+import { DatasetService } from "../../../../service/user-dataset/dataset.service";
+import { FileUploadItem } from "../../../../type/dashboard-file.interface";
+import { Dataset, DatasetVersion } from "../../../../../../common/type/dataset";
+import { untilDestroyed } from "@ngneat/until-destroy";
+import { NotificationService } from "../../../../../../common/service/notification/notification.service";
+
+@Component({
+  selector: "texera-user-dataset-version-creator",
+  templateUrl: "./user-dataset-version-creator.component.html",
+  styleUrls: ["./user-dataset-version-creator.component.scss"],
+})
+export class UserDatasetVersionCreator implements OnInit {
```

Comment:
Rename `UserDatasetVersionCreator` to `UserDatasetVersionCreatorComponent`.

Reply:
Done